lib/recurly/Http.js

```diff
@@ -1,9 +1,17 @@
 const https = require('https')
+const zlib = require('zlib')
 const casters = require('./Caster')
+const networkErrors = require('./network_errors')
 
 const BINARY_TYPES = [
   'application/pdf'
 ]
+const TIMEOUT = 60000
+
+const zlibOptions = {
```

Comment:
We have a convention of capitalizing const globals. Could we change this to `ZLIB_OPTIONS`?